packages/x-adapter-platform/src/mappers/__tests__/response/top-clicked-response.mapper.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { PlatformTopClickedResponse } from '../../../types/response.types';
+import { topClickedResponseMapper } from '../../response/top-clicked-response.mapper';
+// eslint-disable-next-line max-len
+import { platformTopClickedResponse } from '../../../__tests__/__fixtures__/platform-top-clicked.response';
+
+describe('top clicked response mapper tests', () => {
+  it('should map the top clicked', () => {
+    const response: PlatformTopClickedResponse =
```

Comment:
Why we need to cast this? I can see that the `platformTopClickedResponse` is not PlatformTopClickedResponse type:

```
export const platformTopClickedResponse = {
  topclicked: {
    content: [
      {
        image: 'https://assets.empathy.co/images-demo/31335.jpg',
        gender: 'Men',
```

should we add the type there and fix the type? Because right now the PlatformTopClickedResponse has a PlatformResult as type for the content and it doesnt have some of the values that the response is returning, like color, gender, year...etc